python/cuspatial/cuspatial/utils/gis_utils.py

```diff
@@ -35,5 +35,5 @@ def pip_bitmap_column_to_binary_array(polygon_bitmap_column, width):
     """Convert the bitmap output of point_in_polygon
     to an array of 0s and 1s.
     """
-    binary_maps = apply_binarize(polygon_bitmap_column.data_array_view, width)
+    binary_maps = apply_binarize(polygon_bitmap_column.data_array_view(mode="read"), width)
     return binary_maps
```

Comment:
```
with acquire_spill_lock():
    return apply_binarize(...)
```

Or, I guess assign to the temporary and then return.

Reply:
done 👍